packages/mui-material/src/ListItem/ListItem.js

```diff
@@ -423,9 +423,7 @@ ListItem.propTypes /* remove-proptypes */ = {
   /**
    * The components used for each slot inside.
    *
-   * This prop is an alias for the `slots` prop.
-   * It's recommended to use the `slots` prop instead.
-   *
+   * @deprecated use the `slots` prop instead. This prop will be removed in v7. [How to migrate](/material-ui/migration/migrating-from-deprecated-apis/).
```

Comment:
May I ask you to update to the latest copy for the link (How to migrate). For example: https://github.com/mui/material-ui/pull/42179/files#diff-11871663eff3ef5b4247ef4a213a1103e7426bca63d1ac26acef759f045c362aR196

Reply:
Done